src/utils/format_result.ts

```diff
@@ -3,14 +3,18 @@ import { RuntimeName } from '../runtimes/runtime'
 
 import { removeUndefined } from './remove_undefined'
 
-type FunctionResult = Omit<FunctionArchive, 'runtime'> & { runtime: RuntimeName; schedule?: string }
+type FunctionResult = Omit<FunctionArchive, 'runtime'> & {
```

Comment:
did something change here? why did prettier reformat this? 🤔

Reply:
No, but I had added a new property and then removed it, which I guess made Prettier change the formatting.